server/server.go

```diff
@@ -888,13 +892,15 @@ func (s *Server) Start() error {
 		s.ProjectCmdOutputHandler.Handle()
 	}()
 
-	server := &http.Server{Addr: fmt.Sprintf(":%d", s.Port), Handler: n}
+	tlsConfig := &tls.Config{GetCertificate: s.GetSSLCertificate, MinVersion: tls.VersionTLS12}
+
+	server := &http.Server{Addr: fmt.Sprintf(":%d", s.Port), Handler: n, TLSConfig: tlsConfig}
```

Comment:
@camillsir please see this golangci-lint error

Reply:
Ah, I could set ReadHeaderTimeout to a constant or is it better to make it configurable?

Reply:
It's probably fine for now. Thank you.